### ISSUE TITLE
Can we open the kubelet to collect the container_cpu_schedstat_run_seconds_total, container_cpu_schedstat_runqueue_seconds_total and  container_cpu_schedstat_run_periods_total  metrics ？?

### ISSUE THREAD

**Issue by yan234280533** [kubernetes/kubernetes#107216]
### What would you like to be added?

kubele to collect 

 container_cpu_schedstat_run_seconds_total, 
container_cpu_schedstat_runqueue_seconds_total,
container_cpu_schedstat_run_periods_total 

metrics ?

### Why is this needed?

cpu_schedstat_run_seconds_total,
container_cpu_schedstat_runqueue_seconds_total 
container_cpu_schedstat_run_periods_total 

which can respond the container whether be effected in kernel schedule

**Comment by CatherineF-dev** [comment]
Hi Yan, 
these metrics can be scraped from cAdvisor. https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md

Or you want to these metrics to be added to [Summary API](https://github.com/kubernetes/kubernetes/blob/7d309e0104fedb57280b261e5677d919cb2a0e2d/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go)?

**Comment by ehashman** [comment]
I don't think we want to expand the set of metrics exported by cadvisor. If these are not already available in the kubelet, we recommend you run a standalone cadvisor to collect them, as @CatherineF-dev has suggested.

/close
